### PR TITLE
Added empty queue notification

### PR DESCRIPTION
**Pull request by unclebanks**
* Added notification for when queue is empty.

### CODE REVIEW

**Comment by RedSparr0w** [comment]
There doesn't seem to be any sound files in this PR?

**Comment by unclebanks** [comment]
Sorry about that Red, I did these changes on the same fork that I'm doing map changes on so I deleted/re-forked and forgot the sounds file. :/

File: src/scripts/breeding/Breeding.ts [hunk 1]
```diff
@@ -266,6 +266,15 @@ class Breeding implements Feature {
             if (this.queueList().length) {
                 const nextEgg = this.createEgg(this.queueList.shift());
                 this.gainEgg(nextEgg);
+                if ((this.queueSlots() - this.queueList().length) === this.queueSlots()) {
```
Comment (RedSparr0w) [inline src/scripts/breeding/Breeding.ts:269]: Not sure as I haven't fully tested, but aren't we able to simplify this by just checking if the queue is now empty?
```suggestion
                if (!this.queueList().length) {
```

Comment (unclebanks) [reply src/scripts/breeding/Breeding.ts:269]: If you're saying so I am sure we can. I'll commit it now.

**Review by RedSparr0w — APPROVED** [review]
Looks good to me, Thanks for the PR!

**Pull request merged**